src/django/contricleaner/lib/serializers/row_serializers/row_additional_ids_serializer.py

```diff
@@ -0,0 +1,79 @@
+import re
+from contricleaner.constants import AdditionalIDs
+from contricleaner.lib.serializers.row_serializers.row_serializer import (
+    RowSerializer,
+)
+
+
+class RowAdditionalIdsSerializer(RowSerializer):
+    __DUNS_RE = re.compile(r'^\d{9}$')
```

Comment:
What about validation of hyphens for DUNS ids?

> A DUNS number is sometimes formatted with embedded dashes to promote readability, such as "15-048-3782"

from https://en.wikipedia.org/wiki/Data_Universal_Numbering_System

Reply:
Thanks @VadimKovalenkoSNF.

But one of the further sentences is:
> The dashes are not part of D&B's official definition of the DUNS number.

I highlighted this point in one of the conversations with Stephan, and we decided not to accept dashes in the DUNS values at this stage.

cc @protsack-stephan

Reply:
Ack.